README.rst

```diff
@@ -103,114 +103,69 @@ Alternatively, you can explicitly start and end a span:
+  different exporters to send the traces to. There are three built-in exporters,
+  which are ``PrintExporter``, ``FileExporter`` and ``LoggingExporter``, the
+  other exporters are provided as `extensions <#trace-exporter>`__.
 
-    exporter = file_exporter.FileExporter(file_name='traces')
-    tracer = context_tracer.ContextTracer(exporter=exporter)
+* **Sampler**, which determines how telemetry data got sampled.
+  By default is using ``AlwaysOnSampler``, the other options are
+  ``AlwaysOffSampler`` and ``ProbabilitySampler``
 
-Propagators
-~~~~~~~~~~~
+* **Propagator**, which is in charge of serializing and deserializing the
+  ``SpanContext`` and its headers. The default propagator is
+  ``TraceContextPropagator``, the rest options are ``BinaryFormatPropagator``,
+  ``GoogleCloudFormatPropagator`` and ``TextFormatPropagator``.
 
-You can specify the propagator type for serializing and deserializing the
-``SpanContext`` and its headers. The default propagator is
-``TraceContextPropagator``, the rest options are ``BinaryFormatPropagator``,
-``GoogleCloudFormatPropagator`` and ``TextFormatPropagator``.
 
-This example shows how to use the ``GoogleCloudFormatPropagator``:
-
-.. code:: python
-
-    from opencensus.trace.propagation import google_cloud_format
-
-    propagator = google_cloud_format.GoogleCloudFormatPropagator()
-
-    # Deserialize
-    span_context = propagator.from_header(header)
-
-    # Serialize
-    header = propagator.to_header(span_context)
-
-This example shows how to use the ``TraceContextPropagator``:
+You can customize while initializing a tracer.
 
 .. code:: python
 
     import requests
 
     from opencensus.trace import config_integration
-    from opencensus.trace.propagation.trace_context_http_header_format import TraceContextPropagator
-    from opencensus.trace.tracer import Tracer
+    from opencensus.trace import file_exporter
+    from opencensus.trace import tracer as tracer_module
+    from opencensus.trace.propagation import google_cloud_format
+    from opencensus.trace.samplers import probability
 
     config_integration.trace_integrations(['httplib'])
-    tracer = Tracer(propagator=TraceContextPropagator())
+
+    tracer = tracer_module.Tracer(
+        exporter=file_exporter.FileExporter(file_name='traces'),
+        propagator=google_cloud_format.GoogleCloudFormatPropagator(),
+        sampler=probability.ProbabilitySampler(rate=0.5),
+    )
 
     with tracer.span(name='parent'):
         with tracer.span(name='child'):
             response = requests.get('http://localhost:5000')
 
-Blacklist Paths
-~~~~~~~~~~~~~~~
-
-You can specify which paths you do not want to trace by configuring the
-blacklist paths.
-
-This example shows how to configure the blacklist to ignore the ``_ah/health`` endpoint
-for a Flask application:
+You can use configuration file for Flask/Django/Pyramid. For more information,
+please read the individual integration documentation `here <#integration>`_.
```

Comment:
Why not link "individual integration documentation"?

Reply:
I thought there are multiple documents, but I like your idea. Updated.